tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -393,11 +402,12 @@ def __validate_case_closure_ftl(self, t4, t5s, error_msg):
                 t4_record,
                 "COUNTABLE_MONTH_FED_TIME",
                 line_num,
-                error_msg
+                error_msg,
+                deprecated=True
```

Comment:
confirmed the following no longer appears on the error report ✔️

`At least one person who is head-of-household or spouse of head-of-household on case must have Item 26 (Number of Months Countable Toward Federal Time Limit) >= 60 since Item 9 (Reason for Closure) = 03: federal 5 year time limit.`